src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -2170,6 +2227,8 @@ def x25519_generate_key(self):
         return _X25519PrivateKey(self, evp_pkey)
 
     def x25519_supported(self):
+        if self._fips_enabled:
```

Comment:
These are all things that are going to need to be updated once FIPS-186-5 is standardized. I'm extremely frustrated that this is require to require so much ongoing maintenance.